draft-ietf-scitt-architecture.md

```diff
@@ -1293,3 +1294,60 @@ transparent-statement:sha-256:base64url:5i6UeRzg1...qnGmr1o
 data:application/cose;base64,SGVsb...xkIQ==
 ~~~
 {: #example-transparent-statement-data-url align="left" title="Example Transparent Statement Data URL"}
+
+# Signing Statements Remotely
+
+Statements, such as digital artifacts or structured data regarding artifacts, can be too large or too sensitive to be send to a remote Transparency Services over the Internet.
+In these cases a statement can also be hash, which becomes the payload included in COSE to-be-signed bytes.
+A Signed Statement (cose-sign1) MUST be produced from the to-be-signed bytes according to {{Section 4.4 of RFC9052}}.
+
+~~~aasvg
+
+ .----+-----.
+|  Artifact  |
+ '----------'
+    |   |
+    v   |               .
+ .--+-------.          / \
+| Statement  +--+     /   \
+ '----------'   |    /     \     .----------.
+        |       +-->+  OR   +-->+  Payload   |
+        v       |    \     /     '---+------'
+ .------+---.   |     \   /          |
+|    Hash    +--+      \ /           |
+ '----+-----'           '            |
+                                     |
+
+           ...  Producer Network ...
+
+                      ...
+
+           ...   Issuer Network ...
+
+                                     |
++-----------+                        |
+| Identity  |     (iss, x5t)         |
```

Comment:
As a "document", should this have rounded corners?
I have time this afternoon to reivew and make PRs to tweak before approving.

Reply:
This box represents a concept for an identity document, not a concrete identity document such as a certificate.

I don't intend to update the PR, but you are welcome to.

Reply:
As the Identity Document pre-exists outside of that data flow, I'd be okay with keeping that box different.